src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -1,6 +1,59 @@
 .contactmomenten {
   margin-top: var(--spacing-giant);
 
+  .contact-form {
```

Comment:
This needs to be taken outside of .contactmomenten or else the disabled button won't work for the other forms.